tiledb/bioimg/converters/scale.py

```diff
@@ -0,0 +1,272 @@
+import multiprocessing
+from enum import Enum
+from functools import partial
+from multiprocessing import Pool
+from typing import List, Tuple
+
+from skimage.transform import resize
+
+import tiledb
+
+from .axes import Axes
+from .tiles import iter_tiles
+
+
+class ScaleMethod(Enum):
+    NEAREST = 0
+    LINEAR = 1
+
+
+class ScalerMode(Enum):
+    NON_PROGRESSIVE = 0
+    PROGRESSIVE = 1
+    CHUNKED_NON_PROGRESSIVE = 2
+    CHUNKED_PROGRESSIVE = 3
+
+
+class Scaler(object):
+    def __init__(
+        self, base: tiledb.array.DenseArray, axes: Axes, scale_factor: List[int]
```

Comment:
- `tiledb.Array` for annotating TileDB arrays, don't care if it's dense or sparse (same comment for all `tiledb.array.DenseArray` occurrences)
- `Sequence` instead of `List` for immutable sequences (same comment for all `List` occurrences)